core/state_prefetcher.go

```diff
@@ -89,6 +89,6 @@ func precacheTransaction(msg types.Message, config *params.ChainConfig, gaspool
 	// Update the evm with the new transaction context.
 	evm.Reset(NewEVMTxContext(msg), statedb)
 	// Add addresses to access list if applicable
-	_, err := ApplyMessage(evm, msg, gaspool)
+	_, err := ApplyMessage(evm, msg, gaspool, nil)
```

Comment:
Do we need to pass nil everywhere? '...' (ellipsis) doesn't require that.

Reply:
Resolved ✅